src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -21,7 +21,10 @@
         "NETStandard.Library": {
           "version": "1.6.1",
           "exclude": "runtime"
-        }
+        },
+        "Microsoft.NETCore.Platforms": "1.1.0",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0",
+        "Microsoft.NETCore.ILAsm": "2.0.0-beta-25027-03",
```

Comment:
This mismatch between ILAsm and CoreCLR version is writing a `publish` warning to stderr, failing the auto-update build: https://devdiv.visualstudio.com/DevDiv/_build?buildId=591416. I can disable fail on stderr, but is this really necessary?